### PR TITLE
add model ouput to kcmas

### PR DESCRIPTION
**Pull request by justadogistaken**
Emitting model inference result of pod to kcmas. Which could help centric components leverage the output to work on optimization stuff.

### CODE REVIEW

File: pkg/agent/sysadvisor/plugin/inference/inference.go [hunk 1]
```diff
@@ -127,3 +163,106 @@ func (infp *InferencePlugin) fetchModelResult(ctx context.Context) {
 
 	wg.Wait()
 }
+
+func (infp *InferencePlugin) syncModelResults(ctx context.Context) {
```
Comment (waynepeking348) [inline pkg/agent/sysadvisor/plugin/inference/inference.go:167]: not suggest to emit kcmas-related metrics everywhere, maybe it's better to put in metric package instead

Comment (justadogistaken) [reply pkg/agent/sysadvisor/plugin/inference/inference.go:167]: I agree with you, but why I emit kcmas-metrics is it's hard to reuse podSyncer under metric package.
So do you accept that is goes as:
Create a new file under metric package. And define an exported function(not belongs to any struct). Then call it under `InferencePlugin.Run`?

**Review by sun-yuliang — APPROVED** [review]

File: pkg/agent/sysadvisor/plugin/inference/models/borwein/types/types.go [hunk 2]
```diff
@@ -33,4 +33,37 @@ type BorweinParameter struct {
 // the first key is the pod UID
```
Comment (csfldf) [inline pkg/agent/sysadvisor/plugin/inference/models/borwein/types/types.go:33]: move key description to Results property?

File: pkg/agent/sysadvisor/plugin/inference/inference.go [hunk 3]
```diff
@@ -68,14 +74,25 @@ func NewInferencePlugin(pluginName string, conf *config.Configuration, extraConf
 		return nil, fmt.Errorf("nil emitterPool")
 	}
 
-	emitter := emitterPool.GetDefaultMetricsEmitter().WithTags("advisor-inference")
+	metricEmitter := emitterPool.GetDefaultMetricsEmitter().WithTags("advisor-inference")
+
+	dataEmitter, err := emitterPool.GetMetricsEmitter(metricspool.PrometheusMetricOptions{
```
Comment (csfldf) [inline pkg/agent/sysadvisor/plugin/inference/inference.go:79]: it seems that dataEmitter isn't used in inferencePlugin?

Comment (justadogistaken) [reply pkg/agent/sysadvisor/plugin/inference/inference.go]: forgot to delete that.
removed already.

**Review by sun-yuliang — APPROVED** [review]

**Review by nightmeng — APPROVED** [review]

**Review by luomingmeng — APPROVED** [review]

**Review by zzzzhhb — APPROVED** [review]

**Review by csfldf — APPROVED** [review]

**Review by waynepeking348 — APPROVED** [review]

**Pull request merged**